tests/test_dashboard_page.py

```diff
@@ -25,16 +27,32 @@ def test_footer_all_elements_visible_and_clickable_tc_002(self, locator):
         assert driver.is_visible(locator)
         assert driver.is_clickable(locator)
 
-    @pytest.mark.parametrize('locator', DashboardPageLocators.locators_dashboard_all)
+    def test_menu_selector_can_be_push_tc_005(self):
```

Comment:
Could you please clarify what you mean under "can_be_push"?

Reply:
done